### ISSUE TITLE
Recursive span and laggy long text

### ISSUE THREAD

**Issue by chaosarium** [HugoFara/lwt#8]
**To Reproduce**
Using the same text, I got different-looking HTMLs in two different versions of LWT (2.2.0 and the original 1.6.31):

A long French text on 1.6.31 looks like this. Each word gets a `span` on its own.

<img width="1414" alt="normal spans" src="https://user-images.githubusercontent.com/38693485/152810998-d15945f2-02d4-46c0-bf73-e3927fa942b1.png">

The same text on 2.2.0 looks like this, with `span`s inside the previous one, making a long chain.

<img width="1576" alt="nesting spans" src="https://user-images.githubusercontent.com/38693485/152810978-2f057a3d-34e0-478b-8959-15cca71b1042.png">

**Expected behavior**
Long texts don't lag out

**Desktop**

- OS: MacOS 12.0.1
- Browser: Safari
- PHP 7.4.12
- MySQL 5.7.32

**Comment by HugoFara** [comment]
Oh, so that's why this are so laggy! I thought I solved the "recursive span" bug long ago but apparently it came back at some point.

I think I have spotted the problem: in ``do_text_text.php``, ``$hideuntil``, ``$cnt``, ``$sid`` are declared in ``wordProcessor``, but should be declared in ``mainWordLoop`` because those information should be passed through different words.

Version 2.2.1 should solve things!
